WORKSPACE

```diff
@@ -1,10 +1,9 @@
-'''
+"""
 This file lists and imports all external dependencies needed to build Oppia Android.
-'''
+"""
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Load statements should be in lexicographical order

Lint failure we get without this fix
```WORKSPACE:6: out-of-order-load: Load statement is out of its lexicographical order. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#out-of-order-load)
```

Reply:
LGTM.